LineApp/src/main/java/org/example/models/PullRequest.java

```diff
@@ -0,0 +1,210 @@
+package org.example.models;
+
+import jakarta.persistence.*;
+
+import java.io.Serial;
+import java.time.ZonedDateTime;
+
+@Entity
+@Table(name = "PullRequest")
+public class PullRequest extends Model{
+    @Serial
+    private static final long serialVersionUID = 3460490108006801863L;
+
+    @Column(name = "repo", nullable = false)
+    private String repo;
+    @Column(name = "base", nullable = false)
+    private String base;
+    @Column(name = "number", nullable = false)
+    private int number;
+    @Column(name = "author_id", nullable = false)
+    private int author_id;
+    @Column(name = "author", nullable = false)
+    private String author;
+    @Column(name = "created_at", nullable = false)
+    private ZonedDateTime created_at;
+    @Column(name = "updated_at", nullable = false)
+    private ZonedDateTime updated_at;
+    @Column(name = "closed_at")
+    private ZonedDateTime closed_at;
+    @Column(name = "merged_at")
+    private ZonedDateTime merged_at;
+    @Column(name = "title", nullable = false)
+    private String title;
+    @Column(name = "body")
+    private String body;
+    @Column(name = "review_comments_url")
+    private String review_comments_url;
+    @Column(name = "commits_url")
+    private String commits_url;
+    @Column(name = "description", nullable = false)
+    private String description;
+    @Column(name = "state", nullable = false, length = 50)
+    private String state;
+
+    public PullRequest(){}
+
+    public PullRequest(String repo, String base, int number,
```

Comment:
Constructor (or any other function or method) that takes more than 3-5 parameters (in this case it's 15!) is a code smell. Don't do that. You can do a couple of things here a) use builder pattern - look it up b) singe you're probably initialize entity object from response json, you can just pass this JSON object as a constructor argument! You also can add helper static constructor method like PullRequest.createFrom(sourceObject..)